examples/05_transformers.ipynb

```diff
@@ -0,0 +1,849 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# constructing the transformer\n",
+    "from sktime.dists_kernels import AggrDist, ScipyDist\n",
+    "\n",
+    "# mean of paired Euclidean distances\n",
+    "my_series_dist = AggrDist(ScipyDist(metric=\"euclidean\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can then evaluate the distance by `transform` or direct call:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# evaluate the metric on two series, via transform\n",
+    "my_series_dist.transform(X_series, X2_series)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# evaluate the metric on two series, by direct call - this is the same\n",
+    "my_series_dist(X_series, X2_series)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# evaluate the metric on two identical panels of three series\n",
+    "my_series_dist(X_panel, X_panel)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this is the same as providing only one argument\n",
+    "my_series_dist(X_panel)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# one series, one panel\n",
+    "# we subset X_panel to univariate, since the distance in question\n",
+    "#     cannot compare series with different number of variables\n",
+    "my_series_dist(X_series, X_panel[[\"var_1\"]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pairwise transformers are composable, and use the familiar `get_params` interface, just like any other `sktime` object and `scikit-learn` estimator:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_series_dist.get_params()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.4 General transformer signature - pairwise transformers<a class=\"anchor\" id=\"section_1_4\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`sktime` also provides functionality for pairwise transformers on tabular data, i.e., mathematical objects of signature `(DataFrame-row, DataFrame-row) -> float`, or, in mathematical notation, $$\\mathbb{R}^n \\times\\mathbb{R}^n\\rightarrow\\mathbb{R}$$.\n",
+    "Common examples are distances between series, or (positive definite) kernels on series.\n",
```

Comment:
i think this line needs to be udpated; curerntly it is just copied from section 1.3
```suggestion
    "Common examples are distances between series, or (positive definite) kernels on series.\n",
```